src/compiler/checker.ts

```diff
@@ -12545,6 +12660,29 @@ namespace ts {
                         }
                     }
                 }
+                else if (target.flags & TypeFlags.Negated) {
+                    if (source.flags & TypeFlags.Negated) {
+                        // ~S is a subset of ~T iff T is a subset of S
+                        return isRelatedTo((target as NegatedType).type, (source as NegatedType).type);
+                    }
+                    if (getObjectFlags(source) & ObjectFlags.FreshLiteral) {
+                        const result = isRelatedTo(source, (target as NegatedType).type);
+                        return result === Ternary.Maybe ? Ternary.Maybe : result ? Ternary.False : Ternary.True;
+                    }
+                    // Relationship check is S ⊂ T

```

Comment:
Do you really need to use this symbol? It almost sounds like a joke, but this could affect memory footprint when bootstrapping the compiler since modern engines can avoid full UTF16 representations https://blog.mozilla.org/javascript/2014/07/21/slimmer-and-faster-javascript-strings-in-firefox/

Reply:
A minification step will simply remove this line. The article suggests to minify the code to utilise the inline strings and also help with strings interning. I guess, v8 should not be so much different in this sense and also win from code minification.